packages/@lwc/perf-benchmarks/scripts/build.js

```diff
@@ -149,10 +147,22 @@ async function processBenchmarkFile(benchmarkFile) {
     await Promise.all([writeHtmlFile(), writeTachometerJsonFile()]);
 }
 
+async function initGlobals() {
+    // Setup hash of benchmark components directory for use in generating the Tachometer json files
+
+    const { hash } = await hashElement(path.join(benchmarkComponentsDir, 'dist'), {
+        algo: 'sha256',
+    });
+    benchmarkComponentsHash = hash;
+}
+
```

Comment:
Rather than a global, I'd prefer to pass it in to `processBenchmarkFile` as an arg, but it's just a taste thing.